tests/unit/test_network_connections.py

```diff
@@ -3723,14 +3723,23 @@ def test_ipv6_dns_with_ipv6_disabled(self):
                 },
             }
         ]
-        self.assertRaisesRegex(
-            ValidationError,
-            "IPv6 needs to be enabled to support IPv6 nameservers.",
-            validator.validate_connection_one,
-            "nm",
-            validator.validate(ipv6_dns_with_ipv6_disabled),
-            0,
-        )
+
+        if sys.version[:3] >= "3.6" and sys.version[3] == ".":
```

Comment:
What does this if statement solve? It seems to replicate a different solution that you added in the past.

Reply:
Please see https://github.com/linux-system-roles/network/blob/main/tests/unit/test_network_connections.py#L3981 how to skip a test.